src/pages/git/git-branch/index.md

```diff
@@ -100,8 +100,9 @@ git branch --help
 man git-branch
 ```
 
-### Other Resources
-- [Git merge](../git-merge/index.md)
-- [Git checkout](../git-checkout/index.md)
-- [Git commit](../git-commit/index.md)
-- [Git stash](../git-stash/index.md)
+### More Information:
+- The `git merge` command: [fCC Guide](https://guide.freecodecamp.org/git/git-merge/)
+- The `git checkout` command: [fCC Guide](https://guide.freecodecamp.org/git/git-checkout/)
+- The `git commit` command: [fCC Guide](https://guide.freecodecamp.org/git/git-commit/)
+- The `git stash` command: [fCC Guide](https://guide.freecodecamp.org/git/git-stash/)
+- Git documentation: [branch man page](https://git-scm.com/docs/git-branch)
```

Comment:
Should this be `branch main page?`

Reply:
Oh, it does look like a typo - it's actually referring to the manual (man) pages, but I can change this (and looks like the stash and gitignore have similar bullets) to just read `Git documentation: branch` or `Git documentation: stash`, etc. so it doesn't look odd. 👍